client/src/featureform/register.py

```diff
@@ -2769,20 +2843,21 @@ def register_postgres(
         )
         ```
         Args:
-            name (str): Name of Postgres provider to be registered
-            description (str): Description of Postgres provider to be registered
-            team (str): Name of team
-            host (str): Internal DNS name of Postgres
-            port (str): Port
-            user (str): User
-            password (str): Password
-            database (str): Database
-            tags (List[str]): Optional grouping mechanism for resources
-            properties (dict): Optional grouping mechanism for resources
+            name (str): (Immutable) Name of Postgres provider to be registered
+            host (str): (Immutable) Hostname for Postgres
+            port (str): (Mutable) Port
+            user (str): (Mutable) User
+            password (str): (Mutable) Password
+            database (str): (Immutable) Database
+            description (str): (Mutable) Description of Postgres provider to be registered
+            team (str): (Mutable) Name of team
+            tags (List[str]): (Mutable) Optional grouping mechanism for resources
```

Comment:
tag and list mutability is confusing because it's always a union of existing tags and props.